### PR TITLE
Add 'working with oauth' docs

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose

Corrected the following .md files:

- invoke-the-oauth-introspection-endpoint.md
- jwt-token-generation.md
- oauth-token-validation-using-soap-service.md
- oauth-transaction-logs.md
- oauth2-token-revocation.md
- oauth2-token-validation-and-introspection.md
- saml2-bearer-assertion-profile-for-oauth-2.0.md
- self-contained-access-tokens.md
- setting-up-oauth-token-hashing.md

Modified related images paths and links

Corrected links in quick-start-guide.md and understanding-what-has-changed.md

Corrected a small grammar mistake in configuring-a-service-provider-for-adaptive-authentication.md

### CODE REVIEW

File: en/docs/using-wso2-identity-server/invoke-the-oauth-introspection-endpoint.md [hunk 1]
```diff
@@ -87,7 +71,7 @@ invoke the OAuth introspection endpoint for the super tenant users.
         `           <IS_HOME>/repository/conf/identity/identity.xml          `
         file.
     
-        ``` xml
+        ``` toml
             <Resource context="(.*)/oauth2/introspect(.*)" secured="true" http-method="all">
```
Comment (gomathyk) [inline en/docs/using-wso2-identity-server/invoke-the-oauth-introspection-endpoint.md]: Is this the toml config?

File: en/docs/using-wso2-identity-server/oauth-transaction-logs.md [hunk 2]
```diff
@@ -2,30 +2,22 @@
...
-        `           Log.Token          ` property to **true**.
     
-
     ``` java
-    <EventListener type="org.wso2.carbon.identity.core.handler.AbstractIdentityHandler"    name="org.wso2.carbon.identity.data.publisher.oauth.listener.OAuthTokenIssuanceLogPublisher" orderId="12" enable="true">
-        <Property name="Log.Token">false</Property>
-    </EventListener>
+    [event.default_listener.oauth_listener]
+    priority= 
+    enable =
```
Comment (gomathyk) [inline en/docs/using-wso2-identity-server/oauth-transaction-logs.md]: aren't we mentioning the values here?

File: en/docs/using-wso2-identity-server/oauth2-token-revocation.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ login and revoke application authorization.
 5.  Once you have logged in, click **View details** under **Authorized
     Apps**. The following page appears, listing out the available
     apps.  
-    ![]( ../../assets/img/103329616/103329617.png)
+    ![oauth-authorized-apps]( ../../assets/img/103329616/oauth-authorized-apps.png)
```
Comment (gomathyk) [inline en/docs/using-wso2-identity-server/oauth2-token-revocation.md]: please correct the path

File: en/docs/using-wso2-identity-server/setting-up-oauth-token-hashing.md [hunk 4]
```diff
@@ -19,35 +16,28 @@ authorization codes.
...
         ```
 
     -   Add the following configuration to specify the algorithm to use
         for hashing:
 
-        ``` xml
-                <HashAlgorithm>SHA-256</HashAlgorithm>
+        ``` toml
+        [oauth]
+        hash_token_algorithm= ""
```
Comment (gomathyk) [inline en/docs/using-wso2-identity-server/setting-up-oauth-token-hashing.md]: Isn't SHA-256 the value?

File: en/docs/using-wso2-identity-server/oauth-transaction-logs.md (hunk 2 above)
Comment (sherenewso2) [reply en/docs/using-wso2-identity-server/oauth-transaction-logs.md]: fixed with commit 8fce9b7e60b0c67e606677f9fc5d06b1612c3ae8

File: en/docs/using-wso2-identity-server/oauth2-token-revocation.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/using-wso2-identity-server/oauth2-token-revocation.md]: fixed with 8fce9b7e60b0c67e606677f9fc5d06b1612c3ae8

File: en/docs/using-wso2-identity-server/setting-up-oauth-token-hashing.md (hunk 4 above)
Comment (sherenewso2) [reply en/docs/using-wso2-identity-server/setting-up-oauth-token-hashing.md]: fixed with 8fce9b7e60b0c67e606677f9fc5d06b1612c3ae8

File: en/docs/using-wso2-identity-server/invoke-the-oauth-introspection-endpoint.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/using-wso2-identity-server/invoke-the-oauth-introspection-endpoint.md]: No. This property is still untemplated. Created this git issue [1] 

[1] https://github.com/wso2/product-is/issues/6132

**Pull request merged**